### PR TITLE
clarify unstable/closed PR warning message

### PR DESCRIPTION
**Pull request by migueldiascosta**

### CODE REVIEW

File: easybuild/tools/github.py [hunk 1]
```diff
@@ -382,7 +382,8 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
 
     if not(pr_data['merged']):
         if not stable or closed:
-            print "\n*** WARNING: Using easyconfigs from unstable/closed PR #%s ***\n" % pr
+            state_msg = "unstable (failed tests or merge conflict)" if not stable else "closed (but not merged)"
```
Comment (boegel) [inline easybuild/tools/github.py]: I think pending tests also result in 'unstable' state?

Comment (migueldiascosta) [inline easybuild/tools/github.py]: sorry, right (that was the original scenario...) 

can we distinguish between pending and failed?

Comment (boegel) [inline easybuild/tools/github.py]: Hmm, we may be able to, but the GitHub API documentation isn't very clear on that...

**Comment by boegel** [comment]
lgtm

**Pull request merged**